reports/hierarchicalrandomeffect_cache/html/__packages

```diff
@@ -6,16 +6,15 @@ tidyr
 ggplot2
```

Comment:
Should this file be deleted from the git repository? It looks like other *_cache/* files are ignored in .gitignore, and perhaps this one snuck in on accident. It does seem to cause an issue as there are packages in the list that may not have been installed on the local machine as they do not appear in the package installer list. This leads to the building of the random effects paper to fail.